forge_tests/helpers/MetadataResolverHelper.sol

```diff
@@ -4,25 +4,25 @@ pragma solidity ^0.8.16;
 import {JBMetadataResolver} from "@juicebox/libraries/JBMetadataResolver.sol";
 
 /**
- * @notice Contract to create Juicebox delegate metadata
```

Comment:
meh, thought I removed the ref to delegates, as it can be used wherever (cf lib natspecs which should be correct then, including little ascii art and stuff)

Reply:
see https://github.com/Bananapus/juice-contracts-v4/pull/37/files#diff-4cefaec2ffaef06618565b1b0a4e1472dfd6937ea0a3471c4ea318b978bfe510L5
can be copy-pasted, as this contract is just to test the lib anyway